includes/discovery/storage/CISCO-FLASH-MIB.inc.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
Could you please rename both files to "cisco-flash.inc.php" instead ?

Reply:
Thanks, it's renamed.